npu.user.js

```diff
@@ -1147,7 +1147,6 @@ var npu = {
 						row.removeClass("npu_completed npu_failed npu_missed");
 						var attended = $("td[n=Attended]", row)[0].attributes["checked"].value == "true";
 						var passed = npu.isPassingGrade($("td:nth-child(13)", row).text().trim());
-						console.log(attended);

```

Comment:
I thought I removed that...
https://github.com/solymosi/npu/pull/3/commits/87e23ce2610afe1596fc7fb048f06591af4d5646